XIVComboVX/XIVComboVX.csproj

```diff
@@ -3,7 +3,7 @@
 	<PropertyGroup>
 		<Product>XIVComboVX</Product>
 		<Authors>VariableVixen, FrigidWalrus, attick, daemitus</Authors>
-		<Version>9.30.0</Version>
+		<Version>9.30.4</Version>
```

Comment:
I use [semantic versioning](https://semver.org/), so this should only go up by one version.

Reply:
Yeah, this was just so I could make new builds. :) I didn't want to bump it up fully, since it's your project.